charts/gardener-extension-shoot-cert-service/templates/crds.yaml

```diff
@@ -209,3 +209,694 @@ spec:
       storage: true
       subresources:
         status: {}
+---
+{{- if .Values.gardener.runtimeCluster.enabled }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Why do we need the CRD already here? Isn't it deployed by the `cert-management` controller deployment routine?

Reply:
The deployment of CRDs by the cert-management controller is disabled. See `charts/internal/internal-cert-management/values.yaml` and `charts/internal/shoot-cert-management-seed/values.yaml`.

```yaml
additionalConfiguration:
- --kubeconfig.disable-deploy-crds
- --source.disable-deploy-crds
- --target.disable-deploy-crds
```

Reply:
Hm, the `issuers` CRD is deployed via `charts/internal/internal-cert-management/templates/crds-v1.yaml`. Can we make this consistent?